components/style/dom.rs

```diff
@@ -746,17 +746,10 @@ pub trait TElement
     /// element-backed pseudo-element, in which case we return the originating
     /// element.
     fn rule_hash_target(&self) -> Self {
-        if let Some(pseudo) = self.implemented_pseudo_element() {
+        if let Some(_) = self.implemented_pseudo_element() {
```

Comment:
Nit: is_some(). Though this was here to try figure out a crash with editor code and we never saw it in the wild.